### PR TITLE
Update to 5.3 and PHP 7.0

### PR DESCRIPTION
**Pull request by morrislaptop**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
-- 5.5
-- 5.6
-- hhvm
+  - 5.6
+  - 7
+
 before_script:
-- composer self-update
-- composer install --prefer-source --no-interaction --dev
+  - pecl install mongodb
```
Comment (timgws) [inline .travis.yml]: mongodb is an optional dependancy. It should not be installed in the test suite (it does not work with HHVM)

File: composer.json [hunk 2]
```diff
@@ -19,15 +19,16 @@
...
+        "illuminate/database": "4.1 - 5.3"
     },
     "suggest": {
         "jenssegers/mongodb": "Use QueryBuilderParser with MongoDB"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.7.*@dev",
-        "mockery/mockery": "^0.9.4"
+        "mockery/mockery": "^0.9.4",
+        "jenssegers/mongodb": "^3.0",
```
Comment (timgws) [inline composer.json]: mongodb is an optional dependancy! It should not be in the required dependancies (it does not work with HHVM, which is supported by this extension)

**Comment by morrislaptop** [comment]
It is in require dev so it is optional

**Comment by morrislaptop** [comment]
Not sure how to get Travis to skip mongo tests just for hhvm

**Comment by timgws** [comment]
@morrislaptop check out this travis.yml: https://github.com/timgws/OpenSkillDatatable/blob/master/.travis.yml

**Comment by morrislaptop** [comment]
i can see you've done this in the main repo so i'll just close this

**Pull request closed without merging**